### PR TITLE
added "reply in gmail" button

### PR DESCRIPTION
**Pull request by ehconitin**
Issue https://github.com/twentyhq/twenty/issues/4217

### CODE REVIEW

File: packages/twenty-front/src/modules/activities/emails/types/MessageChannel.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export type MessageChannel = {
+  __typename: 'MessageChannel';
+  id: string;
+  handle: string;
+  connectedAccountId: string;
+};
```
[the remark this answers is not in the document]
Comment (ehconitin) [reply packages/twenty-front/src/modules/activities/emails/types/MessageChannel.ts:6]: Oh will do that

File: packages/twenty-front/src/modules/activities/emails/right-drawer/hooks/useRightDrawerEmailThread.ts [hunk 2]
```diff
@@ -14,6 +16,11 @@ import { useIsFeatureEnabled } from '@/workspace/hooks/useIsFeatureEnabled';
 export const useRightDrawerEmailThread = () => {
   const viewableRecordId = useRecoilValue(viewableRecordIdState);
   const { upsertRecords } = useUpsertRecordsInStore();
+  const [lastMessageId, setLastMessageId] = useState<string | null>(null);
+  const [lastMessageChannelId, setLastMessageChannelId] = useState<
+    string | null
+  >(null);
+  const [isMessagesFetchComplete, setIsMessagesFetchComplete] = useState(false);
```
[the remark this answers is not in the document]
Comment (ehconitin) [reply packages/twenty-front/src/modules/activities/emails/right-drawer/hooks/useRightDrawerEmailThread.ts:23]: Good suggestion

**Comment by ehconitin** [comment]
@lucasbordeau https://github.com/twentyhq/twenty/pull/6642 got closed on accident because I forced push(learnt hard way), created this new one.

**Comment by lucasbordeau** [comment]
I tested with a fresh import, it works well now, thanks ! 

Yes merging avoids any destructive git operation and that's the preferred way at Twenty.

**Review by lucasbordeau — APPROVED** [review]

**Pull request merged**